### ISSUE TITLE
Enhance Aggregated Report CSV with URL Column

### ISSUE THREAD

**Issue by gagan0123** [GoogleChromeLabs/ps-analysis-tool#850]
## Feature Request

**Description:**
Enhance the existing aggregated `report.csv` report generated by the PSAT CLI to include the URL for each row of cookie data.

**Motivation:**
Currently, the aggregated `report.csv` file provides a summary of cookie counts across all scanned URLs. However, it lacks the context of which URL each row of data corresponds to. This makes it difficult to analyze and compare cookie usage across different websites directly from the aggregated report. 

**User Story:**
As a PSAT CLI user analyzing aggregated reports, I want to see the corresponding URL for each row of cookie data so that I can easily compare cookie usage across multiple websites within a single view.

**Acceptance Criteria:**
- [ ] The downloaded aggregated `report.csv` file includes a "URL" column as the first column.
- [ ] Each row in the `report.csv` file contains the URL of the website the cookie data belongs to.
- [ ] The order of columns to be 
  1. URL
  2. Total Cookies
  3. First Party
  4. Third Party
  5. Analytics
  6. Functional
  7. Marketing
  8. Uncategorized
  9. Cookies Issues
  10. Analytics Issues
  11. Functional Issues
  12. Marketing Issues
  13. Uncategorized Issues

**Additional Information:**
The current format of the aggregated .csv file is:
```
Total Cookies, 34
Total First Party Cookies, 32
Total Third Party Cookies, 2
...
Uncategorized Cookies with Issues, 8
```

The desired format after this feature implementation should be:
```
URL,Total Cookies,First Party,Third Party,Analytics,Functional,Marketing,Uncategorized,Cookies Issues,Analytics Issues,Functional Issues,Marketing Issues,Uncategorized Issues
"Aggregated",46,36,5,2,4,4,24,8,0,0,0,8
"https://example.com",34,32,2,2,4,4,24,8,0,0,0,8
"https://another-example.com",12,10,2,1,3,1,7,3,0,1,0,2 
...
```
This change will significantly improve the usability of the aggregated .csv report for analysis and comparison purposes.

**Comment by gagan0123** [comment]
@amovar18 

Just summarizing the changes made to this feature request,
1. Removed the repeated term "Cookies" from most column titles.
2. Moved the Aggregated total of all columns for the entire list to the 2nd Row instead of being at the last row.

**Comment by gagan0123** [comment]
@amovar18 

Have these changes added to the `develop` branch?

**Comment by maitreyie-chavan** [comment]
Released in [v0.12.0](https://github.com/GoogleChromeLabs/ps-analysis-tool/releases/tag/v0.12.0) 🚀
